src/web/routes/application/confirm.js

```diff
@@ -1,15 +1,26 @@
 const { handleRequestForPath } = require('./middleware')
 const { CONFIRM_URL } = require('./common/constants')
 
-const pageContent = ({ translate }) => ({
-  title: translate('confirm.title'),
-  subTitle: translate('confirm.subTitle')
+const confirmPageContent = req => ({
+  title: req.t('confirm.title'),
+  subTitle: req.t('confirm.subTitle')
+})
+
+const unsuccessfulApplicationPageContent = (req, eligibilityStatus) => ({
+  ...{
+    title: req.t('unsuccessfulApplication.title'),
+    heading: req.t('unsuccessfulApplication.title'),
+    content: req.t(`unsuccessfulApplication.content.${eligibilityStatus}`)
+  }
 })
 
 const getConfirmPage = (req, res) => {
-  res.render('confirm', {
-    ...pageContent({ translate: req.t })
-  })
+  if (req.session.eligibilityStatus === 'ELIGIBLE') {
```

Comment:
`ELIGIBLE` is used in a couple of places, should it be extracted to a constant in `common/constants`?

Reply:
Done.